static/include/components/apis/generated/power_sensor-table.md

```diff
@@ -1,11 +1,11 @@
 <!-- prettier-ignore -->
 | Method Name | Description |
 | ----------- | ----------- |
-| [`GetVoltage`](/components/power-sensor/#getvoltage) | Return the voltage reading of a specified device and whether it is AC or DC. |
-| [`GetCurrent`](/components/power-sensor/#getcurrent) | Return the current of a specified device and whether it is AC or DC. |
-| [`GetPower`](/components/power-sensor/#getpower) | Return the power reading in watts. |
-| [`GetReadings`](/components/power-sensor/#getreadings) | Get the measurements or readings that this power sensor provides. |
-| [`GetGeometries`](/components/power-sensor/#getgeometries) | Get all the geometries associated with the power sensor in its current configuration, in the frame of the power sensor. |
-| [`Reconfigure`](/components/power-sensor/#reconfigure) | Reconfigure this resource. |
-| [`DoCommand`](/components/power-sensor/#docommand) | Execute model-specific commands that are not otherwise defined by the component API. |
-| [`Close`](/components/power-sensor/#close) | Safely shut down the resource and prevent further use. |
+| [`GetVoltage`](/components/power_sensor/#getvoltage) | Return the voltage reading of a specified device and whether it is AC or DC. |
```

Comment:
Does this have the latest script change that fixes this?

Reply:
I think not, you're right! Lemme fix this up r/q.